result/bmpserver.yaml

```diff
@@ -0,0 +1,832 @@
+components:
+  schemas:
+    BmpServers.State.Request:
+      description: >-
+        The request for BMP server information.
+      type: object
+      properties:
+        bmp_server_names:
+          description: >-
+            The names of the BMP Servers  to return results for.
+            An empty list will return results for all BMP Servers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Bmp.Server/properties/name"
```

Comment:
I think this `x-constraint` could be similar as `BmpServer.Metrics.Request` e,g, 
```
          - "/components/schemas/Device.Bmp.ServerV4/properties/name"
          - "/components/schemas/Device.Bmp.ServerV6/properties/name"
```

Reply:
Agreed. Fixed.